puppetdb.rb

```diff
@@ -111,7 +113,7 @@ def build_rsp
 end.parse!
 
 ## handle switches
-if options[:list] || options.empty?
```

Comment:
this change was already made on the live copy of the script so I included it in my commit just to upstream it